includes/class-sensei-course-structure.php

```diff
@@ -828,10 +828,27 @@ function( $a, $b ) use ( $order, $type ) {
 						return 1;
 					}
 
-					return false === $b_position || $a_position < $b_position ? -1 : 1;
+					return false === $b_position || $a_position < $b_position ? - 1 : 1;
 				}
 			);
 		}
+
 		return $structure;
 	}
+
+	/**
+	 * Get first incomplete lesson ID or `false` if there is none.
+	 *
+	 * @return mixed|false
+	 */
+	public function get_first_incomplete_lesson_id() {
```

Comment:
This method is new and can probably be used [for this issue](https://github.com/Automattic/sensei/issues/4493).